openshift-start-configured-cluster

```diff
@@ -128,3 +133,12 @@ oc rsh $KEYCLOAK_POD ./keycloak/bin/kcadm.sh update realms/master \
     --config /tmp/.kcadm.config -r master \
     -s accessTokenLifespanForImplicitFlow=86400 \
     -s accessTokenLifespan=86400
+echo "Creating token review client"
+oc rsh $KEYCLOAK_POD ./keycloak/bin/kcadm.sh create clients \
+    --config /tmp/.kcadm.config -r master \
+    -s clientId=token-review \
+    -s enabled=true \
+    -s publicClient=false \
+    -s "redirectUris=[\"*\"]" \
+    -s "attributes={\"x509.subjectdn\": \"CN=api.keycloak.localhost\"}" \
+    -s clientAuthenticatorType=client-x509
```

Comment:
I do not see an option with the CA that signed the client certs, how are these certs validated ?

Reply:
I'm not sure if I understand, so let me tell you what I know if you'll tell me if that answers your question.

The cert can be validated against the CA using the following command (which passes) `openssl verify -verbose -CAfile ca-bundle.crt  keycloak-client.crt`.

In Keycloak, we configure Wildfly to give us certs [here's the PR](https://github.com/jboss-dockerfiles/keycloak/pull/147/files). If the certificate matches the trust store configured in the server, we receive certificate in Keycloak. Later on, we check if `Subject DN` matches our expectation (inline 143). 

If this doesn't answer your question, please tell me what should I clarify more.